components/Sidebar.tsx

```diff
@@ -411,34 +438,19 @@ export const DocsNav = ({
           </svg>
         </div>
         <div
-          className={classnames('ml-6', { hidden: !active.getStarted })}
-          id='getStarted'
+          className={classnames('ml-6', { hidden: !active.getConcepts })}
+          id='concepts'
         >
-          <DocLink
-            uri='/learn/getting-started-step-by-step'
-            label='Creating your first schema'
-            setOpen={setOpen}
-          />
-          <SegmentSubtitle label='Examples' />
           <div className='pl-4 pb-1 pt-1'>
             <DocLink
-              uri='/learn/miscellaneous-examples'
-              label='Miscellaneous examples'
-              setOpen={setOpen}
-            />
-            <DocLink
-              uri='/learn/file-system'
-              label='Modelling a file system'
-              setOpen={setOpen}
-            />
-            <DocLink
-              uri='/learn/json-schema-examples'
-              label='Other examples'
+              uri='/concepts/glossary'
+              label='JSON Schema glossary'
               setOpen={setOpen}
             />
           </div>
         </div>
       </div>
+
       {/* Reference */}
```

Comment:
I noticed no changes to the Reference section; I understood that this section needed a more intensive overhauling, i.e., by implementing Diataxis.